pkg/endpoint/bpf.go

```diff
@@ -1397,9 +1397,9 @@ func (e *Endpoint) startSyncPolicyMapController() {
 	ctrlName := fmt.Sprintf("sync-policymap-%d", e.ID)
 	e.controllers.CreateController(ctrlName,
 		controller.ControllerParams{
-			Group: syncPolicymapControllerGroup,
+			Group:          syncPolicymapControllerGroup,
+			HealthReporter: e.GetReporter("policy map sync"),
 			DoFunc: func(ctx context.Context) error {
-				// Failure to lock is not an error, it means
```

Comment:
It looks like this part of the comment was removed by accident?

Reply:
Hmm I'll fix that.

Reply:
PR to revert deleted comment line: https://github.com/cilium/cilium/pull/29172